src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -44,8 +45,25 @@ const GET_ADMINS = gql`
 `
 
 const GET_LOGS = gql`
-    query GetLogs ($action: String!,$adminType: String!, $page: Int!, $pageSize: Int!, $sort: String!, $objectType: String! $userId: ID!, $startDateTime: String!, $endDateTime: String!){
-        logs(action:$action, adminType: $adminType, page: $page, pageSize: $pageSize, sort: $sort, objectType: $objectType, userId: $userId, startDateTime: $startDateTime, endDateTime: $endDateTime){
+    query GetLogs ($action: String,
+        $adminType: String, 
+        $page: Int, 
+        $pageSize: Int, 
+        $sort: String, 
+        $objectType: String, 
+        $userId: ID!, 
+        $startDateTime: String, 
+        $endDateTime: String){
+        logs(action:$action, 
+            adminType: 
+            $adminType, 
+            page: $page, 
+            pageSize: $pageSize, 
+            sort: $sort, 
+            objectType: $objectType, 
+            userId: $userId, 
```

Comment:
huh. does Mat, not give you the full user object?

Reply:
nope, ive asked him multiple times about it and hes said its better to just give me the id for some reason